taichi/ir/statements.cpp

```diff
@@ -118,6 +120,7 @@ MatrixPtrStmt::MatrixPtrStmt(Stmt *origin_input,
     element_type() = origin->ret_type.get_element_type();
     element_type().set_is_pointer(true);
   } else {
+    raise(SIGSEGV);
```

Comment:
Why can't we use `TI_ERROR` or C++ exceptions here?

Reply:
shoot, it's a debug info. Let me remove that. Many thanks!